hack/.golint_failures

```diff
@@ -50,7 +50,6 @@ pkg/apis/rbac/validation
 pkg/apis/storage
 pkg/apis/storage/v1
 pkg/apis/storage/v1beta1
-pkg/cloudprovider/providers/fake
```

Comment:
Might have been easier to just add the staging package in here so we can do the golint change in a separate PR :P. I think it's fine though since the golint fixes for fake seem fairly small

Reply:
+1

Reply:
That's a good point, I debated doing it as two PRs but opted for one because I felt bad about moving known untidy code to another package; boyscout rule and all that. If this is too much to review with them combined I can try to split it.

Reply:
`fake` is relatively small so I think it's still reasonable to review :)